kazu/training/modelling.py

```diff
@@ -0,0 +1,213 @@
+from typing import Optional, Union, Tuple
+
+import torch
+from torch import nn
+from transformers import (
+    DistilBertForTokenClassification,
+    BertForTokenClassification,
+    PretrainedConfig,
+    DebertaV2ForTokenClassification,
+)
+from transformers.modeling_outputs import TokenClassifierOutput, BaseModelOutput
+from transformers.utils import check_min_version
+
+check_min_version("4.9.0")  # transformers version check
+
+
+def multi_label_forward(
+    model: nn.Module,
+    outputs: BaseModelOutput,
+    return_dict: bool,
+    device: torch.device,
+    ignore_index: int,
+    labels: Optional[torch.Tensor] = None,
+) -> Union[TokenClassifierOutput, Tuple[torch.Tensor, ...]]:
+    sequence_output = outputs[0]
+    sequence_output = model.dropout(sequence_output)
+    logits = model.classifier(sequence_output)
+    loss = None
+    if labels is not None:
+        loss_fct = nn.BCEWithLogitsLoss()
+        ignore_index_loc = labels.mean(-1).squeeze().int()
+
+        flat_outputs = logits.squeeze()[ignore_index_loc != ignore_index]
+        flat_labels = labels.squeeze()[ignore_index_loc != ignore_index]
+
+        flat_outputs = flat_outputs.to(device)
+        flat_labels = flat_labels.to(device)
+
+        loss = loss_fct(flat_outputs, flat_labels)
+    if not return_dict:
+        output = (logits,) + outputs[2:]
```

Comment:
Why are you doing `outputs[:2]` instead of `outputs[:1]` like in the models you are sub-classing?